### ISSUE TITLE
ankidroid is using unsupported sync protocol

### ISSUE THREAD

**Issue by andrewchenshx** [ankicommunity/anki-devops-services#43]
* **AnkiDroid version number:**
2.15.4

## Summary

Reproduction (what did you do - the more details you give, the better):
click the "sync" button

What was expected to happen:
sync sucessful.

What happened instead (the more details you give, the better):
your client is using unsupported sync protocol(10, supported version:9)

**Comment by RetiredWorld** [comment]
Having same issue here. Deploy via docker.

The docker image's last update date is one year ago, and it become imcompatible with the latest AnkiDroid.

I have to build the image on my own with [this script](https://github.com/ankicommunity/anki-devops-services/blob/develop/scripts/build-devops.sh) to make everything work(AnkiDroid, Anki).

**Comment by andrewchenshx** [comment]
would u pls publish the updated docker image?

**Comment by RetiredWorld** [comment]
> would u pls publish the updated docker image?

Nope, I may not gonna to matain the image, and later it will become outdated once again.

To build your won image the only thing you need to do is to exec the script.

But seems you are also in China, [my article](https://flag.zeka.cloud/2021/7#%E4%BD%BF%E7%94%A8%20Docker%20%E9%83%A8%E7%BD%B2%20anki-sync-server) may help.

Good luck :)

**Comment by andrewchenshx** [comment]
@RetiredWorld , thanks.
the working docker image is not the goal, it's just a tool.
so is it possible to update the existing container directly?
